mono/metadata/sgen-bridge.c

```diff
@@ -593,11 +694,10 @@ sgen_bridge_handle_gc_debug (const char *opt)
 		set_dump_prefix (prefix);
 	} else if (g_str_has_prefix (opt, "bridge-compare-to=")) {
 		const char *name = strchr (opt, '=') + 1;
-		if (init_bridge_processor_by_name (&compare_to_bridge_processor, name)) {
-			if (compare_to_bridge_processor.reset_data == bridge_processor.reset_data) {
```

Comment:
Note, I removed this check. This is probably safe because the comparator is a (very rarely used) debug function